testing/regress/ecl/parquetTypes.ecl

```diff
@@ -0,0 +1,617 @@
+/*##############################################################################
+    HPCC SYSTEMS software Copyright (C) 2024 HPCC Systems®.
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+//class=parquet
+//Cover's data type's supported by ECL and arrow
+
+IMPORT Std;
+IMPORT Parquet;
+
+dropzoneDirectory := Std.File.GetDefaultDropZone();
+
+// ======================== BOOLEAN ========================
+
+booleanRecord := {UNSIGNED testid, STRING3 testname, BOOLEAN value};
+
+booleanDatasetOut := DATASET([
+    {000, 'aaa', TRUE},
+    {001, 'aab', FALSE}
+], booleanRecord);
+
+ParquetIO.Write(booleanDatasetOut, dropzoneDirectory + '/BooleanTest.parquet', TRUE);
```

Comment:
I think rather than having a separate parquetCompression file where it also creates datasets for each type they should be added as an option that can be configure via the regression suite versions. This can be saved for later as it is not crucial.